DigitalLearningSolutions.Web/Views/Supervisor/Shared/_StaffMemberCard.cshtml

```diff
@@ -97,20 +81,15 @@
   </div>
 
   </details>
-  @if (Model.SupervisorDelegateDetail.Confirmed != null)
+  <a class="nhsuk-button nhsuk-u-margin-left-4 nhsuk-u-margin-top-0"

```

Comment:
Since Confirmed has been removed, we should now only show this "View self assessments" link:
`@if (!Model.SupervisorDelegateDetail.CandidateID == null)`

Reply:
Change implemented and committed.